test/ng/snifferSpec.js

```diff
@@ -25,6 +25,25 @@ describe('$sniffer', function() {
     });
 
 
+    it('should be true if running inside nw.js', function() {
```

Comment:
According to https://github.com/angular/angular.js/pull/15633/files#r97973838, `isNw` is just a subcase of detecting whether something is a Chrome Packaged App. In that sense, it makes sense to have it incorporated into the `Chrome Packaged App` test.

In other words, we are not doing anything special if it is `NW`. We have just refined our `Chrome Packaged App`-detection check.